portal/src/graphql/portal/DNSConfigurationScreen.tsx

```diff
@@ -0,0 +1,598 @@
+import React, { useMemo, useContext, useCallback, useState } from "react";
+import { useParams, useNavigate, useSearchParams } from "react-router-dom";
+import cn from "classnames";
+import produce from "immer";
+import { FormattedMessage, Context } from "@oursky/react-messageformat";
+import {
+  Text,
+  DetailsList,
+  IColumn,
+  Stack,
+  SelectionMode,
+  IDetailsListProps,
+  ActionButton,
+  VerticalDivider,
+  TextField,
+  Dialog,
+  IDialogProps,
+  DialogFooter,
+  DefaultButton,
+  MessageBar,
+  MessageBarType,
+  Dropdown,
+} from "@fluentui/react";
+
+import { useAppConfigQuery } from "./query/appConfigQuery";
+import { Domain, useDomainsQuery } from "./query/domainsQuery";
+import { useUpdateAppConfigMutation } from "./mutations/updateAppConfigMutation";
+import { useCreateDomainMutation } from "./mutations/createDomainMutation";
+import { useDeleteDomainMutation } from "./mutations/deleteDomainMutation";
+import { PortalAPIAppConfig } from "../../types";
+import ShowError from "../../ShowError";
+import ShowLoading from "../../ShowLoading";
+import ButtonWithLoading from "../../ButtonWithLoading";
+import { actionButtonTheme, destructiveTheme } from "../../theme";
+import { useDropdown, useTextField } from "../../hook/useInput";
+import {
+  GenericErrorHandlingRule,
+  useGenericError,
+} from "../../error/useGenericError";
+import ErrorDialog from "../../error/ErrorDialog";
+import { clearEmptyObject } from "../../util/misc";
+
+import styles from "./DNSConfigurationScreen.module.scss";
+
+interface DNSConfigurationProps {
+  domains: Domain[];
+  rawAppConfig: PortalAPIAppConfig | null;
+  effectiveAppConfig: PortalAPIAppConfig | null;
+}
+
+interface PublicOriginConfigurationProps {
+  rawAppConfig: PortalAPIAppConfig | null;
+  effectiveAppConfig: PortalAPIAppConfig | null;
+  verifiedDomains: Domain[];
+}
+
+interface DomainListItem {
+  id: string;
+  domain: string;
+  isVerified: boolean;
+  isCustom: boolean;
+}
+
+interface DomainListActionButtonsProps {
+  domainID: string;
+  domain: string;
+  isCustomDomain: boolean;
+  isVerified: boolean;
+  onDeleteClick: (domainID: string, domain: string) => void;
+}
+
+interface DeleteDomainDialogData {
+  domainID: string;
+  domain: string;
+}
+
+interface DeleteDomainDialogProps extends Partial<DeleteDomainDialogData> {
+  visible: boolean;
+  dismissDialog: () => void;
+}
+
+const DOMAIN_LIST_STYLES: IDetailsListProps["styles"] = {
+  headerWrapper: { marginTop: "-10px" },
```

Comment:
This makes the header and details list element overlaps. I think we can just omit the 'Domain List' header and remove this style.

Reply:
removed